dask/dataframe/groupby.py

```diff
@@ -1012,6 +1091,31 @@ def std(self, ddof=1, split_every=None, split_out=1):
         result = map_partitions(np.sqrt, v, meta=v)
         return result
 
+    @derived_from(pd.DataFrame)
+    def cov(self, ddof=1, split_every=None, split_out=1):
```

Comment:
Oddly, the Pandas implementation doesn't support `ddof`.  Do we know if there is a reason for this?

Reply:
NM.  Well, from your tests it looks like it must.  The Pandas docstring comes through a bit oddly on my system.

Reply:
In pandas `cov` is wrapped and handed to Groupby